index.js

```diff
@@ -26,13 +29,23 @@ function loadJsonFile(jsonFilename) {
   }
 }
 
-function composeSourceMaps(tsMap, babelMap, tsFileName, tsContent) {
-  const map = new SourceMapGenerator()
+function composeSourceMaps(tsMap, babelMap, tsFileName, tsContent, babelCode) {
   const tsConsumer = new SourceMapConsumer(tsMap)
+  const useRawMapping = Array.isArray(babelMap)
+  if (useRawMapping) {
+    babelMap = fromRawMappings([
```

Comment:
Can't we just iterate over the raw mappings instead of converting them first? In either case, I think I'd prefer if we make a separate function for composing raw mappings and have just one branch in the body of the main transformer. Keeps the logic cleaner, I think.